src/routes/Home.jsx

```diff
@@ -21,64 +15,18 @@ const HomeContainer = styled(Container)`
   min-height: 100vh;
 `;
 
-const Header = styled.div`
-  margin-bottom: ${spacer(3)};
-  text-align: center;
-
-  svg {
-    width: 20rem;
-  }
-`;
-
-const Heading = styled.h1`
-  color: #fff;
-  font-size: 3.5rem;
-  line-height: 1.4;
-  margin-bottom: ${spacer(3)};
-  margin-top: 0;
-  text-align: center;
-
-  /* stylelint-disable */
-  ${utils.responsive({
-    lg: `
-      font-size: 4rem;
-    `,
-  })};
-  /* stylelint-enable */
-`;
-
 export class Home extends React.PureComponent {
   static propTypes = {
     dispatch: PropTypes.func.isRequired,
-    user: PropTypes.object.isRequired,
-  };
-
-  handleClickLogin = () => {
-    const { dispatch } = this.props;
-
-    dispatch(login());
   };
 
   render() {
-    const { user } = this.props;
+    const { dispatch } = this.props;
 
     return (
-      <Background key="Home" data-testid="HomeWrapper">
-        <HomeContainer verticalPadding>
-          <Header>
-            <Logo />
-          </Header>
-          <Heading>{config.description}</Heading>
-          <Button
-            animate={user.status === 'running'}
-            onClick={this.handleClickLogin}
-            size="xl"
-            textTransform="uppercase"
-            data-testid="Login"
-          >
-            <Icon name="sign-in" />
-            <Text ml={2}>Start</Text>
-          </Button>
+      <Background key="home" data-testid="home">
+        <HomeContainer>
+          <WelcomeView dispatch={dispatch} />
```

Comment:
If you connect the WelcomeView component then you don't need to pass dispatch around like this.

Reply:
let's circle back and talk about how we want to organize/connect this stuff